src/test/cypress/cypress/integration/UserCard.spec.js

```diff
@@ -501,28 +458,21 @@ describe('User Card ', function () {
       it('Cannot edit card from not allowed entity', ()=>{
         cy.loginOpFab('operator2_fr','test');
         cy.get('of-light-card').should('have.length',1);
-        cy.get('of-light-card').eq(0).click()
-          .find('[id^=opfab-feed-light-card]')
-          .invoke('attr', 'data-urlId')
-          .then((urlId) => {
-              cy.hash().should('eq', '#/feed/cards/'+urlId);
-              cy.get('#opfab-card-edit').should('not.exist');
-        });
+        cy.get('of-light-card').eq(0).click();
+        cy.get('of-card-details').find('of-detail');
```

Comment:
At this place it is needed as it permits to wait the rendering of the card detail and to check after the edit button does not exist . 
Otherwise it may be that the button edit is not visible because the rendering of the component has not been done